README.md

```diff
@@ -312,3 +312,159 @@ where:
+accounts:
+  <account_at_origin_chain>:
+    keystore_path: "<origin_account_keystore_file>"
+    keystore_password_path: "<origin_account_password_file>"
+  <account_at_auxiliary_chain>:
+    keystore_path: "<auxiliary_account_keystore_file>"
+    keystore_password_path: "<auxiliary_account_password_file>"
+origin_contract_addresses:
+    erc20_gateway: <erc20Gateway_contract_address>
+facilitate_tokens:
+    - "<supported_tokens>"
+```
+* Create account on origin and auxiliary nodes. At origin, fund avatar account with ETH. At auxiliary, fund avatar account with base coin. Save the keystore files. For each account, create a password file.
+* Replace `<account_at_origin_chain>` with the address generated at origin node.
+* Replace `<account_at_auxiliary_chain>` with the address generated at auxiliary node.
+* Replace `<origin_chain_url>` with endpoint of origin chain.
+* Replace `<auxiliary_chain_url>` with endpoint of auxiliary chain.
+* Replace `<graph-node-ws-endpoint>` with ws endpoint of graph node.
+* Replace `<graph-node-rpc-endpoint>` with rpc endpoint of graph node.
+* Replace `<origin_account_keystore_file>` with path of keystore file of origin account.
+* Replace `<origin_account_password_file>` with path to password file for auxiliary account.
+* Replace `<auxiliary_account_keystore_file>` with path to keystore file of auxiliary account.
+* Replace `<auxiliary_account_password_file>` with path to password file for auxiliary account.
+* Replace `<erc20Gateway_contract_address>` with address of erc20gateway contract address at origin chain.
+* Replace `<supported_tokens>` with the token address for which facilitation needs to be done. It can be specified in list format supported by yaml.
+
+Example:
+```
+version: v0.14
+architecture_layout: MOSAIC1
+personas:
+  - facilitator
+metachain:
+  origin:
+    avatar_account: "0xcd8b7d0211e51b78d3fd1209ca8a6955fc7dfbca"
+    node_endpoint: https://rpc.slock.it/goerli
+    graph_ws_endpoint: ws://localhost:8000/subgraphs/name/m1_facilitator
+    graph_rpc_endpoint: http://localhost:8000/subgraphs/name/m1_facilitator
+  auxiliary:
+    avatar_account: "0xd14087a083fdd4049dcf1e7eb01ee2f5c89eb1c9"
+    node_endpoint: https://chain.mosaicdao.org/hadapsar
+    graph_ws_endpoint: ws://localhost:8000/subgraphs/name/m1_facilitator
+    graph_rpc_endpoint: http://localhost:8000/subgraphs/name/m1_facilitator
+accounts:
+  "0xcd8b7d0211e51b78d3fd1209ca8a6955fc7dfbca":
+    keystore_path: "0xcD8b7D0211E51B78d3fd1209CA8A6955FC7dFBca.json"
+    keystore_password_path: "0xcD8b7D0211E51B78d3fd1209CA8A6955FC7dFBca.password"
+  "0xd14087a083fdd4049dcf1e7eb01ee2f5c89eb1c9":
+    keystore_path: "0xD14087A083fDd4049dcF1E7Eb01ee2f5C89eb1C9.json"
+    keystore_password_path: "0xD14087A083fDd4049dcF1E7Eb01ee2f5C89eb1C9.password"
+origin_contract_addresses:
+    erc20_gateway: "0x65E1A3d1e95E271325B882aa139997AFAE6F0351"
+facilitate_tokens:
+    - "0x0000000000000000000000000000000000000012"
```

Comment:
Specify that `facilitate_tokens` can be null in case, all tokens needs to be supported